generic-ec-curves/src/lib.rs

```diff
@@ -6,6 +6,7 @@
 //! [`generic-ec` crate]: https://docs.rs/generic-ec
 
 #![cfg_attr(not(test), deny(clippy::unwrap_used, clippy::expect_used))]
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Can't we set no_std unconditionally since we don't use it anywhere?

Reply:
Yup! will do.